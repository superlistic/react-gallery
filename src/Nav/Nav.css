.nav {
  background-color: var(--clr-bg);
  display: grid;
  position: fixed;
  grid-template-columns: 1fr 3fr 1fr;
  align-items: center;
  padding: 2rem;
  justify-content: center;
  width: 100%;
  z-index: 10;
}
.nav__logo {
  justify-self: flex-start;
}
.nav__links {
  /* #D30C7B */
  background-color: var(--clr-bg);
  display: flex;
  /* justify-content: center; */
  justify-self: center;
}

.nav__item {
  margin: 0 1rem;
  color: var(--clr-dark);
  text-decoration: none;
}
