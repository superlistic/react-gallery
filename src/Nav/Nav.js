import React from 'react';
import { Link } from 'react-router-dom';
import '../main.css';
import './Nav.css';

const Nav = () => {
  return (
    <div className="nav">
      <div className="nav__logo">LOGO</div>
      <div className="nav__links">
        <Link className="nav__item" to="/">
          Home
        </Link>
        <Link className="nav__item" to="/about">
          About
        </Link>
        <Link className="nav__item" to="/search">
          Search
        </Link>
      </div>
    </div>
  );
};

export default Nav;
