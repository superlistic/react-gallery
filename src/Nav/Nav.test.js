import Nav from './Nav';
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter, Route, Switch } from 'react-router-dom';

test('renders elements', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Switch>
        <Route exact path="/search">
          <Nav />
          ).toJSON();
        </Route>{' '}
      </Switch>
    </BrowserRouter>
  );

  expect(tree).toMatchSnapshot();
});
