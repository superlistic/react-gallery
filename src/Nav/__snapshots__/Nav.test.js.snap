// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements 1`] = `null`;
